### PR TITLE
sorter: fix unstable unit test

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
fix unstable unit test

### Check List 

Tests 

 - Unit test

### Release note

- No release note

### CODE REVIEW

File: cdc/puller/sorter/sorter_test.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func (s *sorterSuite) TestSorterBasic(c *check.C) {
 	sorter, err := NewUnifiedSorter("/tmp/sorter", "test-cf", "test", 0, "0.0.0.0:0")
 	c.Assert(err, check.IsNil)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
```
Comment (amyangfei) [inline cdc/puller/sorter/sorter_test.go:79]: Is there any CI link that shows this test is unstable?
It seems `FAIL: sorter_test.go:323: sorterSuite.TestSorterIOError` is an unstable case.

Comment (leoppro) [reply cdc/puller/sorter/sorter_test.go:79]: the error I found:
```
[2021-06-01T02:58:29.766Z] FAIL: sorter_test.go:60: sorterSuite.TestSorterBasic
[2021-06-01T02:58:29.766Z] 
[2021-06-01T02:58:29.766Z] sorter_test.go:82:
[2021-06-01T02:58:29.766Z]     c.Assert(err, check.ErrorMatches, ".*context cancel.*")
[2021-06-01T02:58:29.766Z] ... error string = "context deadline exceeded"
[2021-06-01T02:58:29.766Z] ... regex string = ".*context cancel.*"
```

Comment (amyangfei) [reply cdc/puller/sorter/sorter_test.go:79]: Could we estimate how long this test could cost, is this an unknown bug or just slow environment causes slow case.
PTAL @liuzix

Comment (liuzix) [reply cdc/puller/sorter/sorter_test.go:79]: `SorterBasic` timing out is likely due to slow environment. This case is not testing any error handling or edge cases.

Comment (ben1009) [reply cdc/puller/sorter/sorter_test.go:79]: the same case fail at my side several times, is this change ready to merge

**Comment by ben1009** [comment]
the last activity is 22 days ago, @leoppro is this pr still active, do you need help

File: cdc/puller/sorter/sorter_test.go [hunk 2]
```diff
@@ -339,7 +339,7 @@ func (s *sorterSuite) TestSorterIOError(c *check.C) {
 	sorter, err := NewUnifiedSorter("/tmp/sorter", "test-cf", "test", 0, "0.0.0.0:0")
 	c.Assert(err, check.IsNil)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
```
Comment (amyangfei) [inline cdc/puller/sorter/sorter_test.go:342]: This test has been fixed once.

**Comment by amyangfei** [comment]
Close this PR, since we have new rules to track unstable CI, if found unstable cases again, feel free to reopen it.

**Pull request closed without merging**